src/app/themes.tsx

```diff
@@ -6,11 +6,32 @@ import { useServerInsertedHTML } from "next/navigation";
 import React, { useState } from "react";
 import { ServerStyleSheet, StyleSheetManager, ThemeProvider } from "styled-components";
 
-export const currentTheme = {
-    foregroundColor: "#000",
-    backgroundColor: "#DDD",
-    fillColor: "lightgreen",
-    fonts: ["Comic Sans MS", "Comic Sans", "cursive"],
+export const lightTheme = {
+    foregroundColor: "#000000",
+    backgroundColor: "#eeeeee",
+    primaryForegroundColor: "#000000",
+    primaryColor: "#63a036",
+    secondaryForegroundColor: "#ffffff",
+    secondaryColor: "#066940",
+    accentColor: "#609fd3",
+    accentForegroundColor: "#ffffff",
+    surfaceForegroundColor: "#000000",
+    surfaceColor: "#ffffff",
+    errorColor: "#ff624e",
```

Comment:
Better be consistent with the notation of "foreground" vs "background" color
Also maybe arrange them in terms of importance? e.g. primary should be above secondary